docs/content/docs/how-to/creating-tables.md

```diff
@@ -114,6 +114,78 @@ Partition keys must be a subset of primary keys if primary keys are defined.
 By configuring [partition.expiration-time]({{< ref "docs/maintenance/manage-partition" >}}), expired partitions can be automatically deleted.
 {{< /hint >}}
 
+## Create Table As
+
+Tables can also be created and populated by the results of a query. 
+
+{{< tabs "create-table-as" >}}
+
+{{< tab "Flink" >}}
+
+// Flink will create target table first and then start a `INSERT INTO target_table SELECT * FROM source_table` job,
```

Comment:
Please explain the case to use `CREATE TABLE AS` and provide the difference between the usage of this and `CREATE TABLE LIKE`.

Reply:
yes。we should  offer the difference